lib/v8.js

```diff
@@ -368,7 +367,7 @@ class DefaultDeserializer extends Deserializer {
     }
     // Copy to an aligned buffer first.
     const buffer_copy = Buffer.allocUnsafe(byteLength);
-    copy(this.buffer, buffer_copy, 0, byteOffset, byteOffset + byteLength);
+    this.buffer.copy(buffer_copy, 0, byteOffset, byteLength);
```

Comment:
If you use buffer.copy I think you need to keep the `byteOffset +`: `buf.copy(target[, targetStart[, sourceStart[, sourceEnd]]])`